src/values/ident.rs

```diff
@@ -85,7 +107,12 @@ impl<'i> ToCss for DashedIdent<'i> {
   where
     W: std::fmt::Write,
   {
-    dest.write_dashed_ident(&self.0, true)
+    let css_module_custom_idents_enabled = dest
+      .css_module
+      .as_mut()
+      .map_or(false, |css_module| css_module.config.custom_idents);
+
+    dest.write_dashed_ident(&self.0, true, css_module_custom_idents_enabled)
```

Comment:
`write_dashed_ident` already checks the `dashed_idents` option. I'm not sure that the `custom_idents` option also needs to affect dashed idents as well?